api/v1alpha1/loadbalancer_types.go

```diff
@@ -67,6 +67,14 @@ type ConsistentHash struct {
 	// +optional
 	// +notImplementedHide
 	Header *Header `json:"header,omitempty"`
+
+	// The table size for consistent hashing, must be prime number limited to 5000011.
+	//
+	// +kubebuilder:validation:Minimum=2
+	// +kubebuilder:validation:Maximum=5000011
+	// +kubebuilder:default=65537
+	// +optional
+	TableSize *uint64 `json:"tableSize,omitempty"`
```

Comment:
can you also add a comment `// +notImplementedHide`? we can remove this comment after this api has been implemented.

Reply:
Sure.